CONTRIBUTING.md

```diff
@@ -7,7 +7,41 @@ submitting bug reports and feature requests or writing code which can be incorpo
 Anyone can take part in our community and it there are no rules or requirements stopping you from joining.
 Hopefully this document will help you make the jump!
 
-Please read [Development section](https://github.com/ampache/ampache/wiki#development).
```

Comment:
duplicate the updated file in `.github/CONTRIBUTING.md`

Reply:
Not for this PR, but maybe a symlink would work too. I copied it nonetheless.